common/list/lists-en.html

```diff
@@ -408,3 +408,16 @@ <h2 id="dl-inline">Inline description list</h2>
 	&lt;dt>Term&nbsp;:&lt;/dt>
 	&lt;dd>Short description&lt;/dd>
 &lt;/dl></code></pre>
+
+<h2 id="compact">List with <code>compact</code> utility class.</h2>
```

Comment:
Just to add a little bit of context of use.

```suggestion
<h2 id="compact">List with <code>compact</code> utility class.</h2>
<p>Compact list version primarily used for navigation links on navigational page.</p>
```